eng/Version.Details.xml

```diff
@@ -5,102 +5,102 @@
       Source-build uses transitive dependency resolution to determine correct build SHA of all product contributing repos.
       The order of dependencies is important and should not be modified without approval from dotnet/source-build-internal.
     -->
-    <Dependency Name="Microsoft.WindowsDesktop.App.Ref" Version="8.0.0-rtm.23504.7" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
I don't think we want these flowing through interbranch flow. These should be through codeflow. That raises whether codeflow is actually working for 8.0.2xx

Reply:
Ok, figured out why we weren't seeing flow. The default channels weren't configured. Additionally, CI wasn't enabled: https://github.com/dotnet/sdk/commit/7efeafa7eba05a73721a016f931d1ac0f590de23

Let's let 2xx flow to installer and then fix the merge conflicts that will result
CC @Forgind

Reply:
@marcpopMSFT Still do not see [release/ 8.0.2xx] flow in installer repro, is there still other blocking?

Reply:
my PR got merged to enable the channel but builds are still failing. Pinged dnceng.
https://github.com/dotnet/arcade/pull/14119/files

Reply:
https://github.com/dotnet/installer/pull/17548